src/components/Layout/Header/Header.js

```diff
@@ -22,13 +22,14 @@ const Navigation = styled.nav`
   > ul {
     display: flex;
     flex-direction: column;
-    position: fixed;
+    position: absolute;
     transition: transform 0.3s linear;
```

Comment:
btw this one makes the drop down menu to quickly appear on resize

Reply:
I had issues with stacking on more-services page, as categories on top have position sticky. I will address it later. Navigation on desktop and on mobile has different styling, might have to do it separately, so one will be visible only on mobile and other on desktop.